geode-connectors/src/test/java/org/apache/geode/connectors/jdbc/internal/xml/ElementTypeTest.java

```diff
@@ -14,7 +14,6 @@
  */
 package org.apache.geode.connectors.jdbc.internal.xml;
 
-import static org.apache.geode.connectors.jdbc.internal.xml.ElementType.CONNECTION;
 import static org.apache.geode.connectors.jdbc.internal.xml.ElementType.CONNECTION_SERVICE;
```

Comment:
Here's a reference to the product code for element of connection-service (or is it connector-service?).

Reply:
Yes we still have the connector-service element. We only removed the connection sub-element of it.